guide/payments/receive.md

```diff
@@ -0,0 +1,215 @@
+
+**QR Codes**
+
+QR Codes are also a common way to exchange payment details when the two devices are near one another. It is not without its faults though, as some people may not be familiar with QR Codes, and lower-end phones may have difficulty reading them.
+
+Here are some constraints to consider when displaying QR Codes:
+
+- Complex QR Codes can be difficult for devices with low-resolution cameras to read.
+- Use high contrast between the QR Code blocks and their background.
+- Display the QR Code at the largest size, so the scanner can detect it easily.
+
+_For more technical details about optimizing   QR Codes [see this article](https://bitcoinops.org/en/bech32-sending-support/#creating-more-efficient-qr-codes-with-bech32-addresses)._
+
+## Awaiting the payment
+Once the payment details are shared, until the sender creates a transaction and broadcast it, there is uncertainty of when the payment would be completed. Think about how to keep the receiver informed while they are waiting.
+
+Similar to [the processes a sender]({{ '/guide/payments/send/#transaction-processing-and-confirmation' | relative_url }}) sees, with the inclusion of this waiting period, here is what the receiver can expect:
+
+[WHAT DOES THE RECEIVER SEE?]
+
+If your application has an area where the user can see a list of payment requests, this would be a good place to use it to also indicate which specific stage the payment is in.
+
+Then once the payment has been finalized, you should also consider what the receiver may want to do with those funds. You may want to help facilitate those follow-up activities e.g. moving the funds to a shared multi-key wallet or doing a [coinjoin]({{ '/guide/glossary/#coinjoin' | relative_url }}).
+
+
+## Address compatibility
+There are currently [three different versions]({{ '/guide/glossary/#address' | relative_url }}) of bitcoin addresses. From newest to oldest they are; Segwit, Script, and Legacy. It is the best practice for applications to support the newest address version, SegWit, as default.
+
+Since "legacy" addresses are still in use[^1], and some older applications have yet to upgrade. Script addresses can be used to resolve incompatibility issues that can arise when the sender's application does not recognize the SegWit address provided by the receiver.
+
+To illustrate the problem, suppose the sender is using a legacy wallet and paying to the receiver's SegWit address. In this case, the sender's wallet may incorrectly warn them that the address is invalid or not supported. This can confuse the sender’s end, leading them to think that the receiver provided an incorrect address.
+
+The receiver should then have the ability to switch to a Script address that does not have all the benefits of SegWit, like cheaper transactions but will be compatible with the sender's wallet.
```

Comment:
```suggestion
The receiver should have the ability generate a script SegWit address that does not have all the benefits of native SegWit, like cheaper transactions, but will be compatible with the sender's legacy wallet.
```

Reply:
So this may not be correct, but I was trying to be consistent with the glossary and other parts of the guide which refer to SegWit (3...) as Script addresses.

Reply:
I feel that leans to the more technical side whereas I see the guide being more for the lay man / designers who may not be so technical